src/main/java/core/basesyntax/model/Penguin.java

```diff
@@ -0,0 +1,28 @@
+package core.basesyntax.model;
+
+import core.basesyntax.interfaces.Swimmable;
+
+import java.util.List;
+
+public class Penguin extends Aves implements Swimmable {
```

Comment:
in your case Penguin can fly, cause it extends Aves which implements swimmable, so let's make Aves not implement flyable (you will do it classes that can fly)